kernelci/runtime/lava.py

```diff
@@ -180,6 +181,16 @@ def get_params(self, job, api_config=None):
         return params
 
     def generate(self, job, params):
+        # if LAVA runtime dont have set notify.callback.token
```

Comment:
And, why set some default callback attributes?  The idea is that if there's no callback or "notify" attribute then the job just doesn't generate a callback when it completes.  Loading API tokens from the environment is not going to work here.  Also, the tokens are stored in the LAVA database anyway.